test.sh

```diff
@@ -8,4 +8,6 @@ mkdir build || true && \
     cmake -G Ninja .. && \
     cmake --build . && \
     cd .. && \
-    mvn test
+    #mvn  '-Dtest=WriterExtraTablesBenchmarkTest*' test
+    # mvn  '-Dtest=TableTest*' test
+    mvn '-Dtest=WriterTest#canInsertNullRows' test
```

Comment:
Euuh, isn't it a change that shouldn't have been committed? It seems as it runs a single test only, nope?

Reply:
You are correct, this script isn't used anywhere in CI, just a convenience script for development. Will undo the change.